packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -41,10 +41,11 @@ let globalCooldownTimeout = null;
  * methods to toggle this state. Ensures only one tooltip is open at a time and controls
  * the delay for showing a tooltip.
  */
-export function useTooltipTriggerState(props: TooltipTriggerProps): TooltipTriggerState {
+export function useTooltipTriggerState(props: TooltipTriggerProps = {}): TooltipTriggerState {
   let {delay = TOOLTIP_DELAY} = props;
   let {isOpen, open, close} = useOverlayTriggerState(props);
   let id = useMemo(() => `${++tooltipId}`, []);
+  let closeTimeout = useRef<ReturnType<typeof setTimeout>>();
```

Comment:
this looks quirky but the return type of `setTimeout` is depending on env and on types loaded for that env so this ensures that it will always work~, regardless of what you are going to put into `tsconfig.json#compilerOptions.lib`